mason/markers/genotyped/external_links.mas

```diff
@@ -0,0 +1,97 @@
+<%doc>
+    Call this page when using an arrayref of marker hashes, where each hash has a key of marker_name
+</%doc>
+
+<%args>
+    $markers => undef
+</%args>
+
+<%init>
+    my @marker_names = ();
+    foreach my $marker (@$markers) {
+        push(@marker_names, $marker->{marker_name});
+    }
+    my @unique_marker_names = do { my %seen; grep { !$seen{$_}++ } @marker_names };
+</%init>
+
+<style>
+    #external-links-list {
+        list-style-type: none;
+    }
+</style>
+
+
+<div id="external-links">
+    <div id="external-links-table"><p><em>Loading...</em></p></div>
+    <div id="external-links-flankingseq"></div>
+</div>
+
+
+<& /instance/external_link_sources.mas &>
```

Comment:
This mason file seems to be missing

Reply:
Thanks for picking that up - it's in our T3 mason repo and not necessary for this.  I'll create a PR to remove it.